ACadSharp/Objects/MultiLeaderAnnotContext.cs

```diff
@@ -0,0 +1,362 @@
+﻿using System.Collections.Generic;
+
+using ACadSharp.Attributes;
+using ACadSharp.Tables;
+
+using CSMath;
+
+
+namespace ACadSharp.Objects
+{
+
+	/// <summary>
+	/// This class represents a subset ob the properties of the MLeaderAnnotContext
+	/// object, that are embedded into the MultiLeader entity.
+	/// </summary>
+	public partial class MultiLeaderAnnotContext : CadObject
+	{
+		public override ObjectType ObjectType => ObjectType.UNLISTED;
+
+		/// <inheritdoc />
+		public override string SubclassMarker => DxfSubclassMarker.MultiLeaderAnnotContext;
+
+		/// <inheritdoc />
+		public override string ObjectName => DxfFileToken.ObjectMLeaderContextData;
+
+
+		/// <summary>
+		/// Leader Roots
+		/// </summary>
+		public IList<LeaderRoot> LeaderRoots { get; } = new List<LeaderRoot>();
+
+
+		/// <summary>
+		/// BD	40	Overall scale
+		/// </summary>
+		[DxfCodeValue(40)]
+		public double OverallScale { get; set; }
+
+		/// <summary>
+		/// 3BD	10	Content base point
```

Comment:
As mentioned in https://github.com/DomCR/ACadSharp/pull/169/files#r1360745917 please remove the types and the codes from the Summary tag.